src/main/java/mate/academy/spring/dto/response/UserResponseDto.java

```diff
@@ -1,22 +1,11 @@
 package mate.academy.spring.dto.response;
 
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
 public class UserResponseDto {
```

Comment:
```suggestion
@Data
@Accessors(chain = true) // this is optional
public class UserResponseDto {
```
using `@Data` for DTO's is fine
you can use just `@Getter` `@Setter` too

Reply:
ok